spec/dummy/db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 32) do
+ActiveRecord::Schema.define(version: 33) do
```

Comment:
:eyes: many tables seems to be deleted from schema, is this intended?

Reply:
Deleted tables were duplicated in schema.

Reply:
🙈